libcnb-cargo/src/cli.rs

```diff
@@ -25,6 +26,9 @@ pub(crate) struct PackageArgs {
     /// Build for the target triple
     #[arg(long, default_value = "x86_64-unknown-linux-musl")]
     pub target: String,
+    /// Directory for packaged buildpacks, defaults to 'libcnb-packaged' in Cargo workspace root
+    #[arg(long)]
+    pub package_dir: Option<PathBuf>,
```

Comment:
I think this new `--package-dir` option needs a changelog entry added?

Reply:
Added in [292155e](https://github.com/heroku/libcnb.rs/pull/583/commits/292155edaf930f754e40429c4716bbdaffdf840a)